app/package.json

```diff
@@ -170,18 +168,13 @@
     "flow-babel-webpack-plugin": "^1.1.1",
     "flow-bin": "^0.113.0",
     "git-revision-webpack-plugin": "^3.0.6",
-    "highcharts": "^8.2.2",
-    "highcharts-react-official": "^3.0.0",
     "history": "^4.10.1",
-    "hoist-non-react-statics": "^3.3.2",
     "html-webpack-plugin": "^3.2.0",
-    "html2plaintext": "^2.1.2",
+    "husky": "^1.3.1",
```

Comment:
`husky` and `lint-staged` were used for the git commit hook so brought them back